netlify/edge-functions/card.tsx

```diff
@@ -1,44 +1,31 @@
-import { ImageResponse } from "@vercel/og";
-import { NextRequest } from "next/server";
-import { getAvatarByUsername } from "lib/utils/github";
-import { fetchContributorPRs } from "lib/hooks/api/useContributorPullRequests";
-import { getRepoList } from "lib/hooks/useRepoList";
-
-/**
- * @params {string} username - username for the requested user
- * @params {number} w - Width of the card
- */
+// @ts-ignore
```

Comment:
I created #1550 to remove the need for `@ts-ignore`s.

Reply:
I'm thinking of resolving this convo here because it seems like it could be non-blocking but I think we should keep #1550 open and try and figure out this issue because it's definitely annoying that it doesn't work

Reply:
Ok, i got it working with some simple modifications to the tsconfig. I also setup an import maps file in the `netlify/edge-functions` folder